numba/tests/test_np_functions.py

```diff
@@ -476,7 +476,7 @@ def swapaxes(a, a1, a2):
     return np.swapaxes(a, a1, a2)
 
 
-def nan_to_num(X, copy=True, nan=0.0):
+def nan_to_num(X, copy=True, nan=0.0, posinf=None, neginf=None):
     return np.nan_to_num(X, copy=copy, nan=nan)
```

Comment:
The `posinf` and `neginf` values are not being passed to `np.nan_to_num` here, which I think is causing the test failures (the actual implementation and the rest of the test code looks like it *ought* to be OK to me)